src/main/java/com/flexrate/flexrate_back/member/api/SignUpController.java

```diff
@@ -0,0 +1,55 @@
+package com.flexrate.flexrate_back.member.api;
+
+import com.flexrate.flexrate_back.member.application.SignupMemberService;
+import com.flexrate.flexrate_back.member.domain.Member;
+import com.flexrate.flexrate_back.member.dto.SignupRequestDTO;
+import com.flexrate.flexrate_back.member.dto.SignupResponseDTO;
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/*
+ * 회원가입 로그인 API 컨트롤러
+ * @since 2025.04.28
+ * @author 윤영찬
+ */
+@RestController
+@RequestMapping("/members")
+public class SignUpController {
+
+    private final SignupMemberService signupMemberService;
+
+
+    @Autowired
+    public SignUpController(SignupMemberService signupMemberService) {
+        this.signupMemberService = signupMemberService;
+    }
```

Comment:
제가 참고하라고 만들어놓은 controller, service 파일에 작성된 코드처럼, `@Autowired` 대신 `@RequiredArgsConstructor`를 사용해주세요! Lombok이 제공하는 @RequiredArgsConstructor를 쓰면 생성자 주입 코드도 자동 생성해주고, 불변성도 보장됩니다. `@Autowired` 어노테이션을 지양하는 이유는 아래 글들을 참고해주세요~

> https://velog.io/@joypeb/Java-Autowired%EB%A5%BC-%EC%A7%80%EC%96%91%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0
> https://hiorio.tistory.com/entry/Autowired%EB%A5%BC-%EC%A7%80%EC%96%91%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0

```suggestion
import lombok.RequiredArgsConstructor;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.*;

/*
 * 회원가입 로그인 API 컨트롤러
 * @since 2025.04.28
 * @author 윤영찬
 */
@RestController
@RequestMapping("/members")
@RequiredArgsConstructor
public class SignUpController {

    private final SignupMemberService signupMemberService;
```